classes/controller/menu/class-blocks-usage-controller.php

```diff
@@ -250,7 +250,7 @@ public function plugin_blocks_report( $type = 'text' ) {
 					<th>' . __( 'Title', 'planet4-blocks-backend' ) . '</th>
 				</tr>';
 				foreach ( $results as $result ) {
-					echo '<tr><td>'.$result->term_id .'</td>';
+					echo  '<tr><td><a href="' . get_permalink( $result->ID ) . '" >' . $result->ID . '</a></td>';
```

Comment:
Looks like a typo here, at place of `term_id`.

Reply:
also for tag pages the public link would be diff. you may try `get_term_link()` here.

Reply:
> Looks like a typo here, at place of `term_id`.

Good catch! (not really needed anymore, because tags without redirect page are not possible anymore right? But good for backwards compatibilty maybe)

> also for tag pages the public link would be diff. you may try get_term_link() here.

Yep, got it now! :)

Reply:
Should be fixed :)